receiver_custom.html

```diff
@@ -474,11 +476,12 @@ <h3 id= "error_description" class ="cast_text"></h3>
                 screenController.showScreen(loadingScreen);
                 break;
               case OO.EVENTS.PLAYBACK_READY:
-                if (autoPlay) {
-                  // Controlled extention of displaying loading screen, currently set to 3 seconds.  
-                  // ATTN: Receiver HAS TO publish CAN_PLAY in case of autoPlay
+                // Controlled extention of displaying loading screen, currently set to 3 seconds.  
+                // ATTN: Receiver HAS TO publish CAN_PLAY in case of autoPlay
+                player.mb.publish(OO.EVENTS.CAN_PLAY) 
```

Comment:
Missing semicolon. Also, it looks like this would fire `CAN_PLAY` twice when `autoplay` is false. Maybe we can remove the `else` below.